src/tasks/misc.ts

```diff
@@ -94,6 +95,29 @@ import { warCleared } from "./level12";
 
 const meatBuffer = 1000;
 
+function Leprecondo(): number[] {
+  const furnitureFound = new Set(get("leprecondoDiscovered", "").split(",").map(Number));
+
+  // Prioritizing familiar weight/Experience, then Meat Find, then random Booze
+  const f1 = furnitureFound.has(21) ? 21 : 0; // Whiskeybed First to prevent overriding anything important
+  const f2 = furnitureFound.has(8) ? 8 : 0; // Karaoke -> overwritten with treadmill for familiar weight
+  const f3 = furnitureFound.has(9) ? 9 : 0; // Treadmill -> exercise, don't care about the food
+  const f4 = furnitureFound.has(13) ? 13 : 0; // Sous vide -> meat% and random food
+
+  return [f1, f2, f3, f4];
+}
+
+function isLeprecondoComplete(): boolean {
+  const installed = get("leprecondoInstalled", "")
+    .split(",")
+    .map(Number)
+    .sort((a, b) => a - b);
+
+  const target = Leprecondo().sort((a, b) => a - b);
+
+  return installed.length === target.length && installed.every((v, i) => v === target[i]);
+}
```

Comment:
Firstly this is wrong because you're losing the order of the desired furniture items. Secondly...

```ts
return get("leprecondoInstalled", "0,0,0,0") == Leprecondo().join(",")
```
??